backend/backend/settings/base.py

```diff
@@ -404,11 +406,6 @@ def get_required_setting(
 # White lists workflow-api-deployment path
 WHITELISTED_PATHS.append(f"/{API_DEPLOYMENT_PATH_PREFIX}")
 
-# White list paths under tenant paths
```

Comment:
@muhammad-ali-e why was this removed? Has it been moved elsewhere?

Reply:
Tested with some scenarios , Here we don't need these list because of django session authentication handled this. Please let me know are there any case that break this changes. @gopalsamyperumal @VikashPratheepan

Reply:
@muhammad-ali-e specifically did you check for these 2 scenarios?
1. GDrive OAuth
1. Accessing documentation with `http://localhost:8000/api/v1/doc/`